c2corg_ui/templates/outing/helpers/view.html

```diff
@@ -304,3 +304,17 @@
     </tbody>
   </table>
 </%def>
+
+<%def name="get_outing_elevation_profile(geometry)">\
+  <div class="view-details-info col-xs-12">
+    <h3 class="heading closed show-phone" ng-click="mainCtrl.animateHeaderIcon($event)"
+        data-toggle="collapse" data-target="#elevation-profile">
+      <span x-translate>Elevation profile</span><span class="glyphicon glyphicon-menu-right"></span>
```

Comment:
You should use ``translate`` instead of ``x-translate``. The latter directive is an alias that should only be used when the content of the element must not be extracted and pushed to Transifex. Which is not what you want here, is it?